site/content/docs/5.3/components/badge.md

```diff
@@ -114,12 +114,12 @@ Use the `.rounded-pill` utility class to make badges more rounded with a larger
 Consider using our [Boosted Tags]({{< docsref "/components/tags" >}}) instead.
 
 <details>
-<summary>See Bootstrap example(s) that are incompatible with Orange Design System.</summary>
+<summary>See Bootstrap examples that are incompatible with Orange Design System.</summary>
 <br>
 {{< design-callout-alert >}}
-This component should not be used because it does not exist in the Orange Design System specifications.
+This variant should not be used because it does not respect in the Orange Design System specifications.
 
-Instead, consider using our Boosted [Tags]({{< docsref "/components/tags" >}}). You can also refer to the [Tags](https://system.design.orange.com/0c1af118d/p/975c09-tags/b/24dde8) guidelines on the Orange Design System website.
+Instead, consider using our [Tags component]({{< docsref "/components/tags" >}}). You can also refer to the [Tags guidelines](https://system.design.orange.com/0c1af118d/p/975c09-tags/b/24dde8) on the Orange Design System website.
```

Comment:
I don't mind the new "Tags component" wording instead of "Boosted Tags". However, there are other occurrences of the old pattern when you're searching for "Boosted [" in the code.